README.md

```diff
@@ -354,7 +354,7 @@ those features first the tests will begin passing unexpectedly for us and return
 this it is necessary to add a `PASS` entry to the `expected_results.txt` of the testsuite in question.
 
 The `UNSUPPORTED` result code means that the requirements for a test are not met and so it will not be run. Currently
-all tests which use the `/BE` or `/clr:pure` options are unsupported.
+all tests which use the `/clr` or `/clr:pure` options are unsupported. Further, the `/BE` option is unsupported for x64.

```

Comment:
Grammar issue: "Further" should be "Furthermore" here (see [StackExchange](https://english.stackexchange.com/questions/142106/further-versus-furthermore-at-the-beginning-of-a-sentence-meaning); my mother was an English teacher so I am *very* picky :joy_cat:). However, because we have a 120-character line length limit, that would cause the line to wrap. "Also" works here and is shorter:
```suggestion
all tests which use the `/clr` or `/clr:pure` options are unsupported. Also, the `/BE` option is unsupported for x64.
```